### ISSUE TITLE
Checked attribute?

### ISSUE THREAD

**Issue by crisward** [jeromegn/slang#23]
Some attributes need to be removed if set to false

```slim
input name="active" checked=false
```
should render to
```html
<input name="active" />
```

This doesn't seem to happen. Any ideas / work arounds for this?

**Comment by crisward** [comment]
My current work around is...
```slim
- if active
     input type="checkbox" name="active" checked="checked"
- else
     input type="checkbox" name="active"
```

**Comment by jeromegn** [comment]
Released as part of 1.6.1, thanks for pointing this out!

Any attribute `=true` will be present, not value. Any attribute `=false` won't be present.
